README.rst

```diff
@@ -67,3 +69,14 @@ There several options to contact the OpenKAT team:
 - `OpenKAT group on Linkedin <https://www.linkedin.com/>`_ (search for OpenKAT)
 - IRC: #openkat on irc.libera.chat
 - `Signal group <https://signal.group/#CjQKIIS4T1mDK1RcTqelkv-vDvnzrsU4b2qGj3xIPPrqWO8HEhDISi92dF_m4g7tXEB_QwN_>`_
+
+Privacy
+=======
+
+OpenKAT is not designed to collect private information, and in does not act on Private information that its finds. Some information considered to be Identifying information Might be collected through one or more of OpenKAT's plugins and subsequently stored, but only if that information was accessible to OpenKAT. Eg, a phone number or emailadres listed on a website might end up being collected in the 'proofs' that OpenKAT collects of It's actions. No Email or Phone number Models are present and as such they won't be processed into Objects by OpenKAT.
```

Comment:
```suggestion
OpenKAT is not designed to collect private information and it does not act on any private information that its finds. Some information considered to be personally identifying information might be collected through one or more of OpenKAT's plugins and subsequently stored, but only if that information was accessible to OpenKAT. E.g. a phone number or email address listed on a website might end up being collected as part of OpenKAT normal data collection. This data might then be stored for a long time because OpenKAT stores 'proofs' of its actions. No email or phone number models are present and as such they won't be processed into objects by OpenKAT.
```